governance/program/src/state/realm.rs

```diff
@@ -20,3 +27,48 @@ pub struct Realm {
     /// Governance Realm name
     pub name: String,
 }
+
+impl IsInitialized for Realm {
+    fn is_initialized(&self) -> bool {
+        self.account_type == GovernanceAccountType::Realm
+    }
+}
+
+/// Deserializes account and checks owner program
+pub fn deserialize_realm(realm_info: &AccountInfo) -> Result<Realm, ProgramError> {
+    deserialize_account::<Realm>(realm_info, &id())
+}
+
+/// Returns Realm PDA seeds
+pub fn get_realm_address_seeds(name: &str) -> Vec<&[u8]> {
```

Comment:
nit: I wonder if it's a better practice to return seed creators as an array instead of a Vec -- not sure there

Reply:
Yeah, I think you are right. I changed the creators to return arrays